catroid/src/main/java/org/catrobat/catroid/common/LookData.java

```diff
@@ -190,16 +197,16 @@ public String getChecksum() {
 		return fileName.substring(0, 32);
 	}
 
-	protected String getPathToImageDirectory() {
+	String getPathToImageDirectory() {
 		return Utils.buildPath(Utils.buildProjectPath(ProjectManager.getInstance().getCurrentProject().getName()),
 				getSceneNameByLookData(), Constants.IMAGE_DIRECTORY);
```

Comment:
Yes, I would vote for that - the helper methods should also include sounds and the backpack.